### ISSUE TITLE
Support for Python3.8?

### ISSUE THREAD

**Issue by GreatLeaderTechnus** [nateshmbhat/pyttsx3#132]
Hi, I was wondering if there will be support for this library for Python 3.8? I love this library, but it's hard to get 3.7 to work on linux these days.

**Comment by clort81** [comment]
here's what i get with 3.8
```
python3
Python 3.8.5 (default, Aug  2 2020, 15:09:07) 
[GCC 10.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pyttsx3
>>> engine = pyttsx3.init()
Traceback (most recent call last):
  File "/home/clort/.local/lib/python3.8/site-packages/pyttsx3/__init__.py", line 20, in init
    eng = _activeEngines[driverName]
  File "/usr/lib/python3.8/weakref.py", line 131, in __getitem__
    o = self.data[key]()
KeyError: None
```

**Comment by huntyboy102** [comment]
Just wondering if there is an update to this? My IDE says pyttsx3 doesn't exist because it's not compatible. Thank you

**Comment by ike** [comment]
I am also interested in fixing this, I might do some digging in the next couple weeks if that's okay? Seems like this is the only thing holding back from using 3.8.

**Comment by theTijesuni** [comment]
I have the same problem as huntyboy102

**Comment by nateshmbhat** [comment]
https://github.com/nateshmbhat/pyttsx3/issues/136#issuecomment-680098995

**Comment by nateshmbhat** [comment]
P> This is regression in Python 3.7.6 and 3.8.1. Fixed in 3.7.7 and 3.8.2
> 
> Please refert to [Python Issue 38368](https://bugs.python.org/issue38368) for further information.
